content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -0,0 +1,66 @@
+#
+# 
+#
+
+module Transformation
+  module TransformationHost
+    module Common
+      class Utils
+        def initialize(handle = $evm)
+          @debug = true
+          @handle = handle
+        end
+
+        def get_transformation_hosts(ems)
+          ems.hosts.select { |host| host.tagged_with('mf_transformation_host', 'true') }
+        end
+        
+        def get_runners_count_by_host(host)
+          return @handle.vmdb(:vm).all.select { |v| v.custom_get('Transformation Host') == host.name }.size
```

Comment:
@fdupont-redhat dont need the return here

Reply:
@fdupont-redhat This can be an expensive call fetching all the vms and then fetching/comparing host name.
Its good in the sense that it doesn't require locking the host record to add/remove vms being transformed

Reply:
If you have a less expensive approach, I take.

Reply:
@fdupont-redhat You could lookup the CustomAttribute records that has the matching `name` and `value` properties and get the count from there.  If needed you can limit the result set to just VMs by only returning ones with `resource_type  => VmOrTemplate`.

Maybe `@handle.vmdb(:custom_attribute).where(:name => 'Transformation Host', :value => host.name).size`